### ISSUE TITLE
Environment variable becomes "unchecked" if duplicated

### ISSUE THREAD

**Issue by poisa** [a85/POSTMan-Chrome-Extension#3542]
## App Details:
```
Postman for Mac
Version 5.2.0
OS X 16.7.0 / x64
```
## Issue Report:
 1. Did you encounter this recently, or has this bug always been there:
Just encountered this in the last version (don't know if it was there before)

 2. Expected behaviour:
Send the same request multiple times without having to go to the Environment panel and enable (or select) the desired variable.

 3. Console logs (http://blog.getpostman.com/2014/01/27/enabling-chrome-developer-tools-inside-postman/ for the Chrome App, View->Toggle Dev Tools for the Mac app):
Using the Mac app. Can't find the logs.

 4. Screenshots (if applicable). Steps to reproduce the problem

Fortunately I can reproduce this every single time.

First, got to the Environment variables and enter the same variable **twice**. Check one and leave the other unchecked (I don't know if the order makes a difference). In my case I entered the variable "url" and added "http://api.com/api/v1/auth/login" (I replaced my host with api.com for this example). 

Set the request to POST and add the following test. The test is critical because if I don't have a test running, the problem does not happen.

```javascript
tests["Status code is 200"] = responseCode.code === 200;

var jsonData = JSON.parse(responseBody);
tests["Has token"] = _.has(jsonData, 'token')

postman.setEnvironmentVariable("token", jsonData.token);
```
Send the request. It goes out without problems.

Send the request again. An error message appears saying "Could not get any response". There was an error connecting to **%7B%7Burl%7D%7D/api/v1/auth/login**. As you can see the {{url}} string has been url-encoded instead of being replaced. If you go the environment now, you can see that *both* url variables have become unchecked.

**Comment by prashantagarwal** [comment]
@poisa Thanks for reporting this. We are looking into this issue.

**Comment by prashantagarwal** [comment]
@poisa We have fixed this bug internally. This will be fixed in next release.

**Comment by prashantagarwal** [comment]
@poisa This issue is fixed in 5.3.0. Can you please update your app and verify if you still facing this issue. Feel free to reopen this is so.

**Comment by Zuhayer** [comment]
This issues was bugging me irksomely. Thanks for reporting @poisa and thanks for fixing @prashantagarwal. 

This issue is resolved for me in version 5.3.0.
